src/test/EigenPod.t.sol

```diff
@@ -58,10 +58,49 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
     bytes32[] withdrawalFields;
     bytes32[] validatorFields;
 
+
+    // EIGENPODMANAGER EVENTS
+    /// @notice Emitted to notify the update of the beaconChainOracle address
+    event BeaconOracleUpdated(address indexed newOracleAddress);
```

Comment:
Is it possible to import the events like in the examples here: https://book.getfoundry.sh/cheatcodes/expect-emit

Reply:
can't sort this out, not sure it's possible how you're picturing -- the foundry docs are unclear to me.
if you can sort this out then go right ahead.

Reply:
not a big deal, just will error out if we forget to propagate.